File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/messaging/MyClientServerMessage.cs
using System;

namespace com.db4odoc.f1.messaging
{
	class MyClientServerMessage 
	{
		private string _info;

		public MyClientServerMessage(String info)
		{
			this._info = info;
		}
    
		public override String ToString()
		{
			return "MyClientServerMessage: " + _info;
		}
	}
}

File: db4on/reference/vb/db4o-reference-chapters/src/com/db4odoc/f1/messaging/MyClientServerMessage.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Imports System

Namespace com.db4odoc.f1.messaging
    Class MyClientServerMessage
        Private _info As String

        Public Sub New(ByVal info As String)
            Me._info = info
        End Sub

        Public Overrides Function ToString() As String
            Return "MyClientServerMessage: " + _info
        End Function
    End Class
End Namespace

